spec/cypress/integration/schools/AddParticipants.feature

```diff
@@ -6,13 +6,91 @@ Feature: School leaders should be able to manage participants
     And I am logged in as existing user with email "school-leader@example.com"
     And I am on "2021 school participants" page
 
-  Scenario: Should be able to add a new participant
     When I click on "link" containing "Add participants"
-    Then "page body" should contain "What type of participant do you want to add to this cohort?"
+    Then I should be on "2021 school participant type" page
+    And the page should be accessible
+    And percy should be sent snapshot called "school participant type page"
 
+  Scenario: Should be able to add a new ECT participant
     When I click the submit button
     Then "page body" should contain "Please select type of the new participant"
 
     When I set "new participant type radio" to "ect"
     And I click the submit button
-    Then "page body" should contain "Enter their personal details"
+    Then I should be on "2021 school participant details" page
+    And the page should be accessible
+    And percy should be sent snapshot called "school participant etc details page"
+
+    When I click the submit button
+    Then "page body" should contain "can't be blank"
+
+    When I type "James Bond" into field labelled "Full name"
+    And I type "james.bond.007@secret.gov.uk" into field labelled "Email"
+    And I click the submit button
+    Then I should be on "2021 school choose etc mentor" page
+    And the page should be accessible
+    And percy should be sent snapshot called "school participant choose mentor page"
+
+    When I click the submit button
+    Then "page body" should contain "can't be blank"
+
+    When I set "new participant mentor radio" to "51223b41-a562-4d94-b50c-0ce59a8bb34d"
```

Comment:
Really minor, but in future I think I would prefer using the :name over the :value. We're trying to keep this human readable

Reply:
Very much agree - actually I'd prefer to write it as `When I click on <name>` to properly simulate user interaction...